Triangulation_3/include/CGAL/Triangulation_3.h

```diff
@@ -6594,7 +6068,7 @@ template < class Gt, class Tds, class Lds >
 void
 Triangulation_3<Gt,Tds,Lds>::
 _make_big_hole_3D(Vertex_handle v,
-                  std::map<Vertex_triple,Facet>& outer_map,
+                  Vertex_triple_Facet_map& outer_map,
                   std::vector<Cell_handle>& hole,
                   std::vector<Vertex_handle>& vertices,
                   std::map<Vertex_handle, REMOVE_VERTEX_STATE>& vstates)
```

Comment:
Are these maps ever big, and we could gain from using unordered maps.

Reply:
Small or big, I think a flat map would be a better implementation than `std::map`. But I did not want to mix too many different modifications in this PR.